.github/workflows/xcframework.yml

```diff
@@ -0,0 +1,117 @@
+---
+name: xcframework builder
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - "main"
+permissions:
+  attestations: write
+  contents: read
+  id-token: write
+jobs:
+    macos:
+        runs-on: macos-latest
```

Comment:
shall we ping the version ?

Reply:
I don't mind if this bumps to the latest version of  macos / xcode.